cmake/dependencies/AMReX.cmake

```diff
@@ -53,6 +53,16 @@ macro(find_amrex)
 
         set(AMReX_SIMD "${WarpX_SIMD}" CACHE INTERNAL "")
 
+        if(WarpX_FASTMATH OR ABLASTR_FASTMATH)
+            set(AMReX_FASTMATH ON CACHE INTERNAL "")
+            # TODO: set consistently (default: ON)
+            # set(AMReX_CUDA_FASTMATH ON CACHE INTERNAL "")
+        else()
+            # TODO: set consistently (default: ON)
+            # set(AMReX_CUDA_FASTMATH ON CACHE INTERNAL "")
```

Comment:
Just to double check, the CUDA change to do here would be to intentionally enable this even when both `WarpX_FASTMATH` and `ABLASTR_FASTMATH` are `OFF` (i.e., to intentionally avoid disabling it), do I understand this correctly?